### PR TITLE
Int 32 karma on ie11 edge

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
### Jira Story

- [Investigate running Karma in IE/Edge INT-32](https://osi-cwds.atlassian.net/browse/INT-32)

## Description
With this configuration change we can run Karma javascript tests in Internet Explorer 11 and Microsoft Edge via Virtual Machine in order to test portability of the intake application in browsers other than Chrome and Firefox.

Screenshots of the config running the 4 browsers:
screen shot 2018-05-02 at 11 12 27 am
screen shot 2018-05-02 at 11 12 54 am

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 
This a configuration change for the tests we will be running. 

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have ran all tests for the project.
- [ ] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: bin/ci-javascript [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 Xvfb :99 -ac +extension RANDR -screen 0 2500x2500x16 &
-DISPLAY=:99 bin/karma
+DISPLAY=:99 bin/karma CI=true
```
Comment (wmitchell-cambria) [inline bin/ci-javascript]: I think for an ENV variable, it needs to be before the command. `DISPLAY=:99 CI=true bin/karma` It looks like the Jenkins build for this PR is still only running Chrome and Firefox.

**Comment by NagaPradeepGajula-osi** [comment]
@wmitchell-cambria All the karma tests didn't pass in IE and Edge. you can see this in the attached screenshots above. The scope of the story is to investigate if we can run karma in IE and Edge. The investigation was success and this PR talks about implementing it.

File: package.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
     "jquery": "3.2.1",
     "jquery-ujs": "^1.2.2",
     "js-yaml": "^3.9.0",
+    "karma-ievms-launcher": "^0.1.2",
```
[the remark this answers is not in the document]
Comment (money) [reply package.json]: Not understanding the context of your question.

File: bin/ci-javascript (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply bin/ci-javascript]: yes the jenkins build is only supposed to run chrome and firefox as we don't have the ievms support in the containers.

Comment (NagaPradeepGajula-osi) [reply bin/ci-javascript]: will make the changes for ENV variable.

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (NagaPradeepGajula-osi) [reply package.json]: yeah it should be. missed it will update.

**Comment by brunoOSI** [comment]
@wmitchell-cambria, we need to have these browsers covered.
IE11 and Edge tests must be added to the karma roundup (at least on the developer computers for now)

We need additional stories to fix those failing tests int the future.
Hopefully at some point we can make CI able to run these as well.

@NagaPradeepGajula-osi I'd ask for a wiki page explaining how to install the needed VMs

And as @wmitchell-cambria said if you want to specify on single browser to speed up your testing cycle you can use `--browsers=<whatever>`

**Comment by NagaPradeepGajula-osi** [comment]
Updated the Git Wiki https://github.com/ca-cwds/intake/wiki/Running-IE-11-or-Edge-in-VirtualBox

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
Updated the wiki on how to use IE11 and Edge. @brunoOSI

File: bin/karma [hunk 3]
```diff
@@ -27,7 +27,7 @@ if compile_assets
   end
 
   newenv  = { 'NODE_PATH' => NODE_MODULES_PATH.shellescape }
-  cmdline = ['./node_modules/.bin/karma', 'start', './karma.conf.js ']
+  cmdline = ['./node_modules/.bin/karma', 'start', './karma.conf.js']
```
Comment (wmitchell-cambria) [inline bin/karma]: Does this lack of space cause any issue when appending further options to the command?

**Pull request merged**